Upstream-Status: Pending
Signed-off-by: Ross Burton <ross.burton@intel.com>

From a495656665cd4a4f97744741a4741eafa621d65b Mon Sep 17 00:00:00 2001
From: Ross Burton <ross.burton@intel.com>
Date: Mon, 11 Jul 2016 16:57:38 +0100
Subject: [PATCH] elf.h: add relocations for Imagination META

Adding EM_METAG but not the relocations means the kernel doesn't compile as it
guards its own declarations on the presence of EM_METAG.
---
 elf/elf.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/elf/elf.h b/elf/elf.h
index b6112d9..69e85a7 100644
--- a/elf/elf.h
+++ b/elf/elf.h
@@ -3677,6 +3677,10 @@ enum
 
 #define R_TILEGX_NUM		130
 
+/* Imagination META relocs */
+#define R_METAG_ADDR32		2
+#define R_METAG_NONE		3
+
 /* BPF specific declarations.  */
 
 #define R_BPF_NONE		0	/* No reloc */
-- 
2.8.1

